### PR TITLE
fix some more HTML violations in posted_date_widget.tpl

### PR DESCRIPTION
**Pull request by rabuzarus**

### CODE REVIEW

File: view/templates/posted_date_widget.tpl [hunk 1]
```diff
@@ -30,21 +30,23 @@ function dateSubmit(dateurl) {
...
+		</li>
 		{{/foreach}}
 		{{if $cutoff}}
+		</ul>
 		</div>
 		<ul class="datebrowse-ul">
 			<li onclick="showHideDates(); return false;" id="posted-date-collapse" class="fakelink tool">{{$showmore}}</li>
 		</ul>
-		{{/if}}
+		{{else}}
```
Comment (annando) [inline view/templates/posted_date_widget.tpl]: Are you sure with the ```else```?

File: view/templates/posted_date_widget.tpl [hunk 2]
```diff
@@ -30,21 +30,23 @@ function dateSubmit(dateurl) {
...
 	</ul>
+	{{/if}}
```
Comment (annando) [inline view/templates/posted_date_widget.tpl]: To which ```if``` does this ```/if``` belong?

Comment (rabuzarus) [reply view/templates/posted_date_widget.tpl]: to `{{if $cutoff}}`

File: view/templates/posted_date_widget.tpl (hunk 1 above)
Comment (rabuzarus) [reply view/templates/posted_date_widget.tpl]: What do you mean with "Are you sure"?

This template (originally taken from hubzilla) is very confusing. If we reach the `$cutoff_year` the original `ul` is closed and a new `div` + a new `ul` is opened. We need to close it at the end. If there is no `$cutoff` we just need to close the original `ul`.

I did several test + html validator tests. And after 2h of testing and try to understand the code I think this PR is the solution.

Comment (annando) [reply view/templates/posted_date_widget.tpl]: Yeah, the ```<ul>``` does look balanced - but there is another problem with the ```<div>``` that is unbalanced.

**Pull request merged**